xarray/tests/test_computation.py

```diff
@@ -789,6 +813,45 @@ def func(x):
     assert_identical(expected, actual)
 
 
+def test_corr(da):
```

Comment:
Can we do the same thing for `cov`? Copy and paste is OK (parameterizing is another option)

Reply:
Sure, I added a TODO asking exactly that question: 
```
# TODO: up to now, cov is only implicitly tested via corr(), is that sufficient?
def cov(da_a, da_b, dim=None):
```

So, we always want explicit testing, right?